cloud/amazon/ec2_vpc_igw.py

```diff
@@ -39,16 +40,13 @@
 # Note: These examples do not set authentication details, see the AWS Guide for details.
 
 # Ensure that the VPC has an Internet Gateway.
-# The Internet Gateway ID is can be accessed via {{igw.gateway_id}} for use
-# in setting up NATs etc.
-      local_action:
-        module: ec2_vpc_igw
-        vpc_id: {{vpc.vpc_id}}
-        region: {{vpc.vpc.region}}
-        state: present
-      register: igw
-'''
+# The Internet Gateway ID is can be accessed via {{igw.gateway_id}} for use in setting up NATs etc.
+module: ec2_vpc_igw
```

Comment:
Disagree - you can set EC2_REGION as an environment variable or as part of your boto profile. I haven't included region in other aws module examples.